src/events/components/ListView/ListEventPlaceholder.tsx

```diff
@@ -0,0 +1,37 @@
+import { faCalendarAlt, faUser } from '@fortawesome/free-regular-svg-icons/';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classnames from 'classnames';
+import React from 'react';
+import { EventTypeEnum, getEventColor, getEventType } from '../../models/Event';
```

Comment:
https://github.com/dotkom/onlineweb-frontend/wiki/(Opinionated)-Structuring-and-ordering-of-imports

Reply:
This is fixed.